tests/e2e/redis/basic-ingest-redis-serving.py

```diff
@@ -28,27 +28,26 @@
 import uuid
 
 FLOAT_TOLERANCE = 0.00001
-PROJECT_NAME = 'basic_' + uuid.uuid4().hex.upper()[0:6]
+PROJECT_NAME = "basic_" + uuid.uuid4().hex.upper()[0:6]
```

Comment:
Why did all of this formatting change? Can you reverse this please. We can have another PR if we need to update formatting.

Reply:
Okay. It changed cause I ran `black` formatting on it.

Reply:
Reverted changes.